eng/common/scripts/Helpers/DotnetTool-Helpers.ps1

```diff
@@ -0,0 +1,24 @@
+function Get-CodeOwnersTool($toolPath, $toolName, $toolVersion, $toolCommandName, $feedUrl)
+{
+  $command = Join-Path $toolPath $toolCommandName
+  # Check if the dotnet tool exsits or not.
+  if (Get-Command $command -errorAction SilentlyContinue) {
+    return $command
+  }
+  if (!(Test-Path $toolPath)) {
+    New-Item -ItemType Directory -Path $toolPath | Out-Null
+  }
+  Write-Host "Installing the $toolName tool under $toolPath... "
+
+  # Run command under tool path to avoid dotnet tool install command checking .csproj files. 
+  # This is a bug for dotnet tool command. Issue: https://github.com/dotnet/sdk/issues/9623
+  Push-Location $toolPath
+  dotnet tool install --tool-path $toolPath --add-source $feedUrl --version $toolVersion $toolName | Out-Null
```

Comment:
It might be worth capturing the output here, so you can log it if you have an error below.